launch/tier4_localization_launch/config/ndt_scan_matcher.param.yaml

```diff
@@ -65,3 +65,9 @@
 
     # regularization scale factor
     regularization_scale_factor: 0.01
+
+    # whether use de-grounded pointcloud calculate score
```

Comment:
```suggestion
    # A flag for using scan matching score based on de-grounded LiDAR scan
```

Reply:
@cyn-liu This is just a little comment, but instead of resolving the conversation by yourself, would you post a commit which addresses the suggestion in a thread? It would be much easier for the reviewer to check if the issue is correctly addressed :smile: 
e.g.: https://github.com/autowarefoundation/autoware.universe/pull/2254#discussion_r1018578029